dataland-frontend/src/components/pages/ApiKeysPage.vue

```diff
@@ -76,11 +76,7 @@
             </template>
           </MessageComponent>
```

Comment:
done

Reply:
Looks good 👍 

I just noticed that we still use our class ".d-center-div" a lot in different components, mainly to put the loading circle icon in the middle of the page.
I think we could use your MiddleCenterDivWrappe.vue in all of these cases, and then drop the ".d-center-div" CSS-class completely from our code.

This would be more consistent.

What do you think?

Reply:
I think of course we should have consistent layout solutions, but maybe in another task.

Reply:
Ok, agree. I will create a point in our refactoring log and close this conversation.

Reply:
Added it to the refactoring log.